api/proto/teleport/legacy/types/types.proto

```diff
@@ -1823,9 +1825,15 @@ message AccessCapabilitiesRequest {
   // RequestableRoles is a flag indicating that we would like to view the list of roles
   // that the user is able to request.
   bool RequestableRoles = 2 [(gogoproto.jsontag) = "requestable_roles,omitempty"];
+  // ResourceIDs is the list of the ResourceIDs of the resources we would like to view
+  // the necessary roles for.
+  repeated ResourceID ResourceIDs = 3 [
+    (gogoproto.jsontag) = "resource_ids,omitempty",
+    (gogoproto.nullable) = false
+  ];
   // SuggestedReviewers is a flag indicating that we would like to view the list of all
   // reviewers which are suggested by the user's roles.
-  bool SuggestedReviewers = 3 [(gogoproto.jsontag) = "suggested_reviewers,omitempty"];
```

Comment:
Same comment as above. You generally want to avoid changing field numbers to maximize compatibility between old/new versions of the software.